pvlib/modelchain.py

```diff
@@ -1068,6 +1076,51 @@ def faiman_temp(self):
     def fuentes_temp(self):
         return self._set_celltemp(self.system.fuentes_celltemp)
 
+    @property
+    def dc_ohmic_model(self):
+        return self._dc_ohmic_model
+
+    @dc_ohmic_model.setter
+    def dc_ohmic_model(self, model):
+        if model is None:
+            self._dc_ohmic_model = self.no_dc_ohmic_loss
+        elif isinstance(model, str):
+            model = model.lower()
+            if model == 'dc_ohms_from_percent':
+                self._dc_ohmic_model = self.dc_ohms_from_percent
+            elif model == 'no_loss':
+                self._dc_ohmic_model = self.no_dc_ohmic_loss
+            else:
+                raise ValueError(model + ' is not a valid losses model')
+        else:
+            self._dc_ohmic_model = partial(model, self)
+
+    def dc_ohms_from_percent(self):
+        """
+        Calculate time series of ohmic losses and apply those to the mpp power
+        output of the `dc_model` based on the pvsyst equivalent resistance
+        method. Uses a `dc_ohmic_percent` parameter in the `losses_parameters`
+        of the PVsystem.
+        """
+        Rw = self.system.dc_ohms_from_percent()
+        if isinstance(self.results.dc, tuple):
+            self.dc_ohmic_losses = tuple(
```

Comment:
Do we want to assign this to `self.results.dc_ohmic_losses`? If yes, we'll need to add it to the `ModelChainResults` class. If not, we should drop `self.` and leave it as a variable that's just used within this function.

Reply:
I think it should be assigned to the results. Otherwise there is no record of what power was taken out of `p_mp` and no way to confirm, test, or just review the losses after the fact.

It looks like `pvwatts_losses` still creates a `self.losses` attribute. Do we want to think about coordinating the storage of those losses with other losses like `dc_ohmic_losses` now and any others that get added later? Or treat each separately?

Reply:
> It looks like pvwatts_losses still creates a self.losses attribute. 

Thanks for catching that. `pvwatts_losses` and `no_extra_losses` should assign values to `ModelChainResults`, and `ModelChain._deprecated_attrs` should include `losses`. We can address that in another issue/pr.

> Do we want to think about coordinating the storage of those losses with other losses like dc_ohmic_losses now and any others that get added later? Or treat each separately?

Good question. I'd lean toward treating them separately for now unless we have a good idea of how the losses models will evolve in the future.

Reply:
Sounds good. I will work on adding `dc_ohmic_losses` to `ModelChainResults` in the next commit.

Reply:
>  I'd lean toward treating them separately for now

I agree. If `losses` attributes pile up in `ModelChainResults` we can consider creating a `Losses` dataclass or other mechanism to group losses together.